android/gradle/libs.versions.toml

```diff
@@ -20,12 +20,12 @@ cql-engine = "2.4.0"
 cql-evaluator = "2.4.0"
 cql-to-elm = "2.4.0"
 dagger-hilt-compiler = "2.45"
-data-capture = "1.0.0-preview15-SNAPSHOT"
+data-capture = "1.0.0-preview18-SNAPSHOT"
 desugar-jdk-libs = "1.1.5"
 easy-rules-jexl = "4.1.0"
 espresso-core = "3.5.1"
 fhir-common-utils = "0.0.10-SNAPSHOT"
-fhir-engine = "0.1.0-beta03-preview10-SNAPSHOT"
+fhir-engine = "0.1.0-beta04-preview3.3-SNAPSHOT"
```

Comment:
just to make sure I understand, are these Google or Ona releases?

Reply:
They are Ona releases.

Reply:
K, if Android FHIR SDK did a release from main today, could we upgrade to it, or do we have custom code, and/or code from a non-main in our releases?

Reply:
Yes, it should be possible to upgrade to the SDK releases once all the unmerged PRs (in our release variant) are included in the official release. We keep track of the unmerged PRs, documented [in the remote-publishing section of the docs](https://docs.opensrp.io/engineering/android-app/developer-setup/publishing-fhir-sdk-artifacts#remote-publishing)

Reply:
Also, NOTE that the `opensrp/android-fhir` master branch is always in sync with the `google/android-fhir` before we publish any of the SDK libraries.

Reply:
k, is there a list of the unmerged PRs somewhere? I can't figure out where to find that

Reply:
Here are the current unmerged changes PR included in the master-release branch https://github.com/opensrp/android-fhir/commit/master-release.

Reply:
ah sweet, thank you!

Reply:
We can schedule an internal discussion together with @ndegwamartin to identify the gaps with the current setup to identify how best the SDK team can support us.